Makefile

```diff
@@ -490,6 +492,11 @@ docs/admin/prometheus.md: scripts/metricsdocgen/main.go scripts/metricsdocgen/me
 	cd site
 	yarn run format:write:only ../docs/admin/prometheus.md
 
+docs/cli/coder.md: scripts/clidocgen/main.go
```

Comment:
Hm.. you should also depend on the `cli` dir, right? Otherwise, the tool won't catch changes there.

Reply:
Yes, I'm not very familiar with `Makefile` so I got a bit confused on this. So I also should add `cli/coder/main.go` to it? or something like `cli/coder` to catch any changes in the coder cli directory? I would love guidance here.

Reply:
I would try `$(shell find ./cli/ -type f) $(shell find ./enterprise/cli/ -type f)`, to catch every change from both directories.

BTW does your script support the enterprise?

Reply:
Good catch! It does not right now but I can do it right after merging this because I have to think better about how I should organize this in the docs 🤔.